src/components/Header/Header.js

```diff
@@ -2,14 +2,18 @@ import React from "react";
 import { Link } from "react-router-dom";
 import css from "./styles.module.scss";
 import cx from "classnames";
+import withProfile from "../../containers/Profile/index";
 
-const renderHeaderUser = ({ name, image: { url } }, onLogout) => (
+const renderHeaderUser = (imageUrl, displayName, location, onLogout) => (
   <div className={css.user_info}>
     <div className="has-text-right is-hidden-mobile">
       <div className={css.user_meta}>
-        <h4 className={cx("title is-4 is-spaced", css.title)}>Hi, {name}</h4>
+        <h4 className={cx("title is-4 is-spaced", css.title)}>
+          Hi, {displayName}
+        </h4>
         <p className={cx("subtitle is-6", css.subtitle)}>
-          <Link to="/profile">Customize your profile</Link> | <span> </span>
+          <Link to="/profile">{location || "Customize your profile"}</Link> |{" "}
+          <span> </span>
           <span onClick={onLogout} className="icon-logout" />
```

Comment:
When running in smaller (mobile) widths, I'm seeing the logout link and header disappear, leaving only the avatar and profile icon link.  Likely want to take care of this in a seprate ticket; but if not fixed, there's no way to logout on mobile devices.

Reply:
![image](https://user-images.githubusercontent.com/194056/33358376-e38b43f2-d48d-11e7-9d8c-92aa05e16895.png)

Reply:
So in previous iteration of this, there was a subsequent logout link in the left navigation to compensate for this, is that not still the case?

Reply:
yes.  it needs to be put back.  was removed for a demo as it was not working. I'll add a task for it and get it back.